.travis.yml

```diff
@@ -30,32 +37,35 @@ env:
         # so Cython is required for testing. If your package does not include
         # Cython code, you can set CONDA_DEPENDENCIES=''
         - CONDA_CHANNELS='http://ssb.stsci.edu/astroconda astropy-ci-extras'
-        - CONDA_DEPENDENCIES='numpy scipy cython matplotlib numba pysynphot mock bokeh h5py sphinx'
+        - CONDA_DEPENDENCIES='numpy scipy cython matplotlib numba pysynphot mock bokeh h5py sphinx pandas lmfit'
         - PIP_DEPENDENCIES='bibtexparser batman-package astroquery'
 
     matrix:
-        - PYTHON_VERSION=2.7 SETUP_CMD='install'
-        - PYTHON_VERSION=2.7 SETUP_CMD='test'
-        - PYTHON_VERSION=2.7 SETUP_CMD='egg_info'
+        # - PYTHON_VERSION=2.7 SETUP_CMD='install'
+        # - PYTHON_VERSION=2.7 SETUP_CMD='test'
+        # - PYTHON_VERSION=2.7 SETUP_CMD='egg_info'
```

Comment:
Do we need to comment these -- I think we can just delete them? Especially since we'll have them recorded in version control history should we ever lose our minds and decide to support Python2 again...

Reply:
True. Let's scrap it.

Reply:
6e73dcd